ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml

```diff
@@ -711,7 +711,7 @@ $defs:
                       description: Custom structured config added under ethernet_interfaces.[name=<interface>] for eos_cli_config_gen.
                       documentation_options:
                         hide_keys: true
-                      $ref: "eos_cli_config_gen#/keys/ethernet_interfaces/items/"
+                      $ref: "eos_cli_config_gen#/keys/ethernet_interfaces/items"
```

Comment:
Are all these modifications forced by the upgrade or is it more for consistency purposes and something we should make reviewer aware of?
asking as in some places the PR removes the leading `/` after `#` but not everywhere

Reply:
there is a pattern - the `/` can be omitted if there is nothing after `#`, otherwise it is required.